packages/donations-app/devServer/controller/validation.js

```diff
@@ -122,6 +122,13 @@ module.exports = function (app) {
           `is achieved`);
       }
 
+      let fundsAvailable = await Utils.getAmountAvailableForProject(project);
+      if (fundsAvailable < totalClaim) {
+        return res.status(400).send(
+          `Cannot claim ${totalClaim / 100} GBP: ` +
+          `Only ${fundsAvailable / 100} GBP donated so far`);
```

Comment:
"donated so far" is misleading as they might be more donated but some of that already claimed
let's change it to "unclaimed"

Reply:
Feel free to merge afterwards